landcover.mss

```diff
@@ -626,6 +626,27 @@
 }
 
 #cliffs {
```

Comment:
That's right. I might call this *reliefs*, or better *reliefsymbols*. Not being my mother tongue, I would greatly appreciate advises here, thanks.